code/modules/research/nanites/nanite_programs/utility.dm

```diff
@@ -258,6 +258,33 @@
 			SEND_SIGNAL(infectee, COMSIG_NANITE_SYNC, nanites)
 			infectee.investigate_log("was infected by spreading nanites by [key_name(host_mob)] at [AREACOORD(infectee)].", INVESTIGATE_NANITES)
 
+/datum/nanite_program/triggered/nanite_sting
+	name = "Nanite Sting"
+	desc = "Stings a random non-host around the host with a barely-visible cluster of nanites, making them a new host. The target will feel it. \
+			If the cluster finds no valid targets, it returns to the original host."
```

Comment:
"Projects a nearly invisible "spike" of nanites that attempts to infect a nearby non-host with a copy of the host's nanites cluster."

No need to give full details about the mechanics, they can be expanded in the wiki.

Reply:
Shortened the description.